scala/datastax-v4/connection-sigv4/src/main/scala/SampleConnectionWithSigv4.scala

```diff
@@ -12,14 +12,12 @@ object SampleConnectionWithSigv4 {
   // and the AWS default credential chain.
   // see https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html.
   def main(args: Array[String]): Unit = {
-    val resultSet = session.execute("select * from system_schema.keyspaces");
-    val rows = resultSet.all().asScala;
-    rows.foreach({ println } );
+    val resultSet = session.execute("select * from system_schema.keyspaces")
+    val rows = resultSet.all().asScala
+    rows.foreach(println)
```

Comment:
The `.toString` value of a row doesn't seem too useful to print out, here. It just prints out e.g. `com.datastax.oss.driver.internal.core.cql.DefaultRow@4015cc95` once for each row you get back.